src/operator/cudnn_deconvolution-inl.h

```diff
@@ -263,6 +264,17 @@ class CuDNNDeconvolutionOp : public Operator {
                                                1,
                                                1,
                                                CUDNN_CROSS_CORRELATION), CUDNN_STATUS_SUCCESS);
+      #else
+      CHECK_EQ(cudnnSetConvolution2dDescriptor(conv_desc_,
+                                               pad_y,
+                                               pad_x,
+                                               param_.stride[0],
+                                               param_.stride[1],
+                                               1,
```

Comment:
Sorry for the delayed response.  Just starting to look at the existing deconvolution code.  Looks like dilation support is not present along the non-cuDNN path as well. This PR doesn't touch deconvolution.